### ISSUE TITLE
reproduce the accuracy

### ISSUE THREAD

**Issue by heruili** [d-acharya/CovPoolFER#6]
I run the code ,but I do not reproduce the accuracy in the paper.first ,the pkl file in the model you provide I always encounter coding problem.second ,when I try  to generate  the pkl file,but I use classify_sfew alaws meet errors about "DLNP-"..... SO I use classify_sfew_incep generate pkl file,but I do not reproduce the accuracy in the paper.I very need your help

**Comment by vBLFTePebWNi6c** [comment]
I have problems trying to reproduce the paper results with pretrained models too. Model4 only gets around 60% accuracy on RAFDB. @d-acharya could you check your code and provide more info about RAFDB folder structure you use. It seems by me that problem may be either in bad pretrained models or mistake in mapping model predictions to class names.

**Comment by vBLFTePebWNi6c** [comment]
@heruili, did you figured out anything about reproducibility issues?

**Comment by d-acharya** [comment]
@heruili, there is probably some mistake in the code. I will clean it up. It will take time.

**Comment by ghost** [comment]
Cannot produce paper results using pretrained Model2 , any possible reasons ?

**Comment by d-acharya** [comment]
If you cannot get the numbers in the paper by using pretrained models, I would try following data: https://drive.google.com/open?id=1eh93I0ndg6X-liUJDYpWveIShLd0ao_x
and make sure following versions are used:
scikit-learn==0.18.1
tensorflow==1.2.0
numpy==1.14.4
Pillow==4.3.0
python 2.7

Different version of pickle or classifier was found to effect reported numbers.

**Comment by ghost** [comment]
Getting same results with these as well, Accuracy is around 42% for model2

**Comment by AAcquier** [comment]
Has someone sorted out this problem of accuracy?

For the SFEW validation set I am getting an accuracy of:
- 44.9% for the base model
- 49.3% for model2
- 50.2%b for model3

All the modules have been updated to the version shown above

**Comment by vBLFTePebWNi6c** [comment]
@AAcquier I was able to reproduce metrics only by using datasets provided by @d-acharya 
It is clear that they differs somehow from the public available ones.

**Comment by AAcquier** [comment]
Where may I be able to get those datasets? Is it be cause he used the work of[ Zhang et al.](https://github.com/kpzhang93/MTCNN_face_detection_alignment)?

**Comment by vBLFTePebWNi6c** [comment]
@AAcquier here: https://drive.google.com/drive/folders/1eh93I0ndg6X-liUJDYpWveIShLd0ao_x

**Comment by AAcquier** [comment]
Thank you
